jenkins.bash

```diff
@@ -114,6 +114,7 @@ EOM
 
 	if [ -d conformance ]
 	then
+		cp -r conformance/cwltool/cwl_${version}/cwltool_${tool_ver} conformance/cwltool/cwl_${version}/cwltool_latest
```

Comment:
Does this work if the directory already exists?

Reply:
No.
I fixed to remove `cwltool_latest` before copying the current conformance result.